atlassian/bitbucket/__init__.py

```diff
@@ -1948,9 +1948,9 @@ def all_branches_permissions(self, project_key, permission_id, repository_slug=N
 
     def _url_branching_model(self, project_key, repository_slug):
         return self.resource_url(
-            "{}/branchmodel/configuration".format(self._url_repo(project_key, repository_slug)),
-            api_root="rest/branch-utils",
-            api_version="1.0",
+            "{}/branchmodel/configuration".format(self._url_repo(project_key, repository_slug, api_root="rest/branch-utils", api_version="1.0")),
+            api_root='',
+            api_version=''
         )
```

Comment:
Test fails because of formating. Following should work:
```python
        return "{}/branchmodel/configuration".format(
            self._url_repo(
                project_key,
                repository_slug,
                api_root="rest/branch-utils",
                api_version="1.0",
            )
        )
```

Reply:
@Spacetown - Better code, too. :-)